docs/rancher/cloud-provider.md

```diff
@@ -30,61 +28,58 @@ In this page we will learn:
-      a service will take an address based upon its namespace pool cidr/range-namespace. These would look like the following:
-        ```YAML
-        $ kubectl get configmap -n kube-system kubevip -o yaml
-
-        apiVersion: v1
-        kind: ConfigMap
-        metadata:
-        name: kubevip
-        namespace: kube-system
-        data:
-        cidr-default: 192.168.0.200/29
-        cidr-development: 192.168.0.210/29
-        cidr-finance: 192.168.0.220/29
-        cidr-testing: 192.168.0.230/29
-        ```                                                                                                                                                                                               
-- dhcp:  A DHCP server is required. The Harvester LoadBalancer controller will request an IP address from the DHCP server of the Kubernetes nodes.
+  
+  ![](assets/vip-pool.png) 
+  
+- dhcp:  A DHCP server is required. The Harvester LoadBalancer controller will request an IP address from the DHCP server.
+
+!!!note
+    It is not allowed to modify the IPAM mode. You can create a new service if you want to do that.
 
 ### Health Checks
-The Harvester load balancer supports TCP health checks. Supported annotations are shown below:
+The Harvester load balancer supports TCP health checks. You can specify the parameters on the Rancher UI if you enable the health check.
 
-| Key | Value | Required | Description |
+  ![](assets/health-check.png)
+
+Another way to specify the parameters is to add the annotations to the service manually. Supported annotations are shown below:
+
+| Annotation Key | Value Type | Required | Description |
 |:---|:---|:---|:---|
 | `cloudprovider.harvesterhci.io/healthcheck-port` | string | true | Specifies the port. The prober will access the address composed of the backend server IP and the port.
 | `cloudprovider.harvesterhci.io/healthcheck-success-threshold` | string | false | Specifies the health check success threshold. The default value is 1. If the number of times the prober continuously detects an address successfully reaches the success threshold, then the backend server can start to forward traffic.
 | `cloudprovider.harvesterhci.io/healthcheck-failure-threshold` | string | false | Specifies the success and failure threshold. The default value is 3. The backend server will stop forwarding traffic if the number of health check failures reaches the failure threshold.
 | `cloudprovider.harvesterhci.io/healthcheck-periodseconds` | string | false |  Specifies the health check period. The default value is 5 seconds.
 | `cloudprovider.harvesterhci.io/healthcheck-timeoutseconds` | string | false | Specifies the timeout of every health check. The default value is 3 seconds.
+
+!!!note
+    The health check port should be a `nodeport`. We will change it as the service port in the future release. Please check [this issue](https://github.com/harvester/harvester/issues/1697) for details.
```

Comment:
```suggestion
    The health check port is set as a `nodeport`. There's [a feature request](https://github.com/harvester/harvester/issues/1697) to change it to a service port and which will be implemented in a future release.
```

Reply:
The note is to tell users that a valid health check port should be a nodeport.

Reply:
thank you, I updated my suggestion accordingly. Let me know if it makes more sense now.